packages/liotad/liotapkg.sh

```diff
@@ -33,18 +33,19 @@
 
 liota_config="/etc/liota/liota.conf"
 package_messenger_pipe=""
+package_responser_pipe=""
 
 if ! [ `ps -ef | grep "liotad.py" | grep -v grep | wc -l` -gt 0 ]
```

Comment:
Not related to this change, but I really wish we'd move to /var/run/liotad.pid and using "kill -0" rather than ps|grep.

Reply:
Yes, but should do this in another pull request.